src/main/java/com/saucelabs/ci/sauceconnect/AbstractSauceTunnelManager.java

```diff
@@ -546,7 +548,158 @@ public Process openConnection(
       }
       final Process process =
           prepAndCreateProcess(
-              username, apiKey, port, sauceConnectJar, options, printStream, sauceConnectPath);
+              username, apiKey, port, sauceConnectJar, options, printStream, sauceConnectPath, false);
+
+      // Print sauceconnect process stdout/stderr
+      new ProcessOutputPrinter(process.getInputStream(), (String x) -> logMessage(printStream, x)).start();
+      new ProcessOutputPrinter(process.getErrorStream(), (String x) -> logErrorMessage(printStream, x)).start();
```

Comment:
AFAIK this creates two threads to stream logs from out and err. That explains disorder:
```
WARNING: the following flags are not supported and will be ignored:
2025/01/14 18:14:35.554373 [control] [INFO] Sauce Connect Proxy version=5.2.2
- extra-info
```